src/VGAudio/Codecs/CriHca/CriHcaPacking.cs

```diff
@@ -57,17 +57,25 @@ public static void PackFrame(CriHcaFrame frame, Crc16 crc, byte[] outBuffer)
             WriteChecksum(writer, crc, outBuffer);
         }
 
-        public static int CalculateResolution(int scaleFactor, int noiseLevel)
+        private static int CalculateResolution(int scaleFactor, int noiseLevel, int version)
         {
             if (scaleFactor == 0)
             {
                 return 0;
             }
             int curvePosition = noiseLevel - 5 * scaleFactor / 2 + 2;
+            //https://github.com/vgmstream/vgmstream/blob/4eecdada9a03a73af0c7c17f5cd6e08518fd7e3f/src/coding/hca_decoder_clhca.c#L1450
+            if (version >= 0x0300 && curvePosition >= 67) return 0; //FIXME
```

Comment:
It seems that `version >= 0x3000` cannot be removed, otherwise HCA 2.0 decoding would break.